app/bundles/DashboardBundle/Tests/Controller/DashboardControllerTest.php

```diff
@@ -111,14 +112,13 @@ protected function setUp(): void
             $this->securityMock
         );
         $this->controller->setContainer($this->containerMock);
-        $this->sessionMock->method('getFlashBag')->willReturn($this->flashBagMock);
+        $this->sessionMock->method('getFlashBag')->willReturn($this->createMock(FlashBagInterface::class));
```

Comment:
I think this should be reverted, because if one would test something with the flashes, should have the same instance in the constructor and session.

Reply:
I was thinking the same thing, but it would require changing signature of all controllers and I'm not convinced this is worth the BC change. Are you?

Reply:
In this particular case you already have a FlashBag injected. So just changing it back to the same instance of the mock would help to avoid confusion.

Reply:
Ok. The tests are passing without this line. Removing it.